### ISSUE TITLE
Support OpenJ9

### ISSUE THREAD

**Issue by SchweinchenFuntik** [beryx/badass-runtime-plugin#56]
Is there support for OpenJ9 ?

Using JDK 13 Adpt openj9, gradle 6.1

console:
```
JVMJ9VM011W Unable to load j9shr29: /home/funtik/Projects/KMany/build/image/KMany-linux/lib/compressedrefs/libj9shr29.so

JVMJ9VM007E Command-line option unrecognised: -Xshareclasses
```

using example: [badass-runtime-example-kotlin-tornadofx](https://github.com/beryx-gist/badass-runtime-example-kotlin-tornadofx)

build.gradle.kts:
```
application {
    // code
    applicationDefaultJvmArgs = listOf(
            "-Xtune:virtualized",
            "-Xshareclasses",
            "-Xshareclasses:name=${project.name}"
    )
}
```

**Comment by siordache** [comment]
The plugin supports virtually any JDK. To test it with OpenJ9, I created a [fork of badass-runtime-example-kotlin-tornadofx](https://github.com/siordache-forks/badass-runtime-example-kotlin-tornadofx) that uses Gradle 6.1 and JDK 13. The plugin produced the right images. The only problem was that the option `-Xshareclasses` is not supported in the OpenJ9 version from adoptopenjdk, so I needed to [remove these settings](https://github.com/siordache-forks/badass-runtime-example-kotlin-tornadofx/commit/599a4cab37f2519b7a18e9f499c0ab25dc4dfb89) from build.gradle.

The images produced by the [Travis build](https://travis-ci.org/github/siordache-forks/badass-runtime-example-kotlin-tornadofx/builds/687893108) are available [here](https://github.com/siordache-forks/badass-runtime-example-kotlin-tornadofx/tree/images).
